benchmarks/sam_shapes.csv

```diff
@@ -0,0 +1,7 @@
+m,k,n
```

Comment:
presumably you mean shapes of matmuls in sam?

Reply:
Yes, SAM vit_b batch size 16 to be precise